lib/ansible/modules/files/replace.py

```diff
@@ -85,6 +85,7 @@
     version_added: "2.4"
 notes:
   - As of Ansible 2.3, the I(dest) option has been changed to I(path) as default, but I(dest) still works as well.
+  - Option I(follow) has been removed in version 2.5, because this module modifies the contents of the file so I(follow=no) doesn't make sense.
```

Comment:
I'd recommend allowing module users to specify "follow=yes" without error (but perhaps a deprecation warning.)  It would  be useful to use a common playbook with multiple versions of ansible.

Reply:
That's a nice suggestion. Will see what I can do about it.

Reply:
I just checked and you can pass the `follow` parameter and Ansible won't throw and error, so I wouldn't change anything here.